src/docfx/restore/LocalPackageResolver.cs

```diff
@@ -0,0 +1,284 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+
+namespace Microsoft.Docs.Build
+{
+    internal class LocalPackageResolver : PackageResolver
```

Comment:
>LocalPackageResolver [](start = 19, length = 20)

should there be only one PackageResolver, which return different kind of Package?